__tests__/app.test.js

```diff
@@ -21,3 +21,24 @@ describe("/api/topics", () => {
             });
     });
 });
+
+describe("/api/articles", () => {
+    test("GET:200 responds with an array of all articles sorted sorted by date in descending order.", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                articles.forEach((article) => {
```

Comment:
if asserting inside a forEach you should check the length of the array first. What would happen if the response was an empty array? 🤔